packages/web-components/src/image/image.options.ts

```diff
@@ -0,0 +1,46 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Border radius
+ * @public
+ */
+export const BorderRadius = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+  xlarge: 'x-large',
+} as const;
+/**
+ * Types for border radius
+ * @public
+ */
+export type BorderRadius = ValuesOf<typeof BorderRadius>;
+
+/**
+ * Image fit
+ * @public
+ */
+export const ImageFit = {
+  none: 'none',
+  center: 'center',
+  contain: 'contain',
+  cover: 'cover',
+  default: 'default',
```

Comment:
@miroslavstastny just to call out another oddity IMO - having "none" here seems odd if this is going to be nullable (meaning we set defaults in CSS). Is there someone we can sync with to understand the intention of having both "none" and "default" as values in the types here? I see that "none" still applies object fit properties, so that makes sense. But for scenarios where "none" is undefined, `default` seems odd to enumerate given it won't do anything.

No action here, just stating that it's confusing from an API standpoint especially if we don't set it.

Reply:
I should have called this out when I first wrote it in response to the Figma spec. It's does seem silly to have both `none` and `default`. The developer should get the default and opt in for `center`, `contain` and `cover`. Same for image fit: square is default, so only `circular` and `rounded` should be available.

Reply:
Or do we have to be so explicit?

Reply:
Any actionable item here?

Reply:
I don't think so but it's odd and want to raise awareness in these cases.